composer.json

```diff
@@ -67,7 +67,7 @@
 				"source": {
 					"url": "https://github.com/Mermade/openapi3-examples",
 					"type": "git",
-					"reference": "master"
+					"reference": "3e8740c4994310a5d6a35d9b19e405862326f149"
```

Comment:
the structure of https://github.com/Mermade/openapi3-examples changed.

So i decided to set the latest commit with the old structure as virtual 1.0.0 package. Is this fine for you?

I will create a new issue to work with the new structure

Reply:
that's fine, thank you.